app/notifications/process_notifications.py

```diff
@@ -321,7 +328,7 @@ def _get_delivery_task(
 
 
 def send_to_queue_for_recipient_info_based_on_recipient_identifier(
-    notification: Notification, id_type: str, id_value: str, communication_item_id: uuid
+    notification: Notification, id_type: str, communication_item_id: uuid
 ) -> None:
```

Comment:
You can't remove this unless you also remove it from the call in `app/notifications/send_notifications::send_notification_bypass_route`. This would cause all of the Comp & Pen notifications to fail. This call must be mocked out in tests or something.